src/box/lua/upgrade.lua

```diff
@@ -654,7 +663,8 @@ end
 --------------------------------------------------------------------------------
 local function create_vsequence_space()
     create_sysview(box.schema.SEQUENCE_ID, box.schema.VSEQUENCE_ID)
-    box.space._vsequence:format(sequence_format)
+    box.space._space:update({box.schema.VSEQUENCE_ID},
+                            {{'=', 7, sequence_format}})
```

Comment:
I'm not sure, but what do you think about named constants, just local in this file? something like `local SPACE_FIELD_FORMAT = 7`?

Reply:
Thought about this too, but decided not to, because:
- It's usually clear from the context what we update, e.g. here it's obvious that we update the 'flags' field of a system space:
  ```Lua
  _space:update({id}, {{'=', 6, flags}})
  ```
- Using a constant may be difficult with update operations involving paths, e.g.
  ```Lua
  _space:update({_priv.id}, {{'=', '[7][4].type', 'scalar'}})
  ```
- We don't use constants in the existing code. Using it for the format field only would be inconsistent. Adding constants for *all* fields would be tedious:
  https://github.com/tarantool/tarantool/blob/ec1af12943a1c4c0052c584068d3fbd81fa050bc/src/box/lua/upgrade.lua#L723-L738